# This service is used to retrieve all existing objects from the
# interoperability server.

# No inputs necessary.

---

# Whether the request was successful.
bool success

# The object IDs.
# The ith element of this array corresponds to the ith element of the objects
# array.
uint64[] ids

# The objects.
# The ith element of this array corresponds to the ith element of the ids
# array.
Object[] objects
